src/overlays/actors/ovl_En_Sob1/z_en_sob1.h

```diff
@@ -3,6 +3,11 @@
 
 #include "global.h"
 #include "overlays/actors/ovl_En_GirlA/z_en_girla.h"
+#include "objects/object_rs/object_rs.h"
+#include "objects/object_zo/object_zo.h"
+#include "objects/object_oF1d_map/object_oF1d_map.h"
+
+#define ENSOB1_LIMB_MAX MAX(MAX((u32)ZORA_LIMB_MAX, (u32)BOMBSHOPKEEPER_LIMB_MAX), (u32)GORON_LIMB_MAX)
```

Comment:
Is there a reason to cast these individually rather than at the end?

Reply:
It suppresses the `compare different enums` warning.
`warning: comparison between ‘enum ZoraLimbs’ and ‘enum BombShopkeeperLimbs’ [-Wenum-compare]`